docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/layout/page_options_macros.twig

```diff
@@ -26,6 +26,9 @@
 {% endif %}
 
 {% set url = drupal_token('current-page:url') %}
+{% set emailSubject = "Information from the National Cancer Institute Web Site "|t %}
+{% set emailBodyOne = "I found this information on www.cancer.gov and I'd like to share it with you: "|t %}
```

Comment:
Check out https://www.drupal.org/node/2047135 for a better way. You should be able to treat the body as one long string.

Reply:
To be clear the .po would look something like:
```
msgid "File usage information for {{ arguments.fid }}"
msgstr "Información de uso de archivos para {{ arguments.fid }}"
```

Because it will translate the string before replacing the placeholders.